webpack/assets/javascripts/react_app/common/grid-breakpoints.scss

```diff
@@ -0,0 +1,6 @@
+$pf-global--breakpoint--xs: 0 !default;
+$pf-global--breakpoint--sm: 576px !default;
+$pf-global--breakpoint--md: 768px !default;
+$pf-global--breakpoint--lg: 992px !default;
+$pf-global--breakpoint--xl: 1200px !default;
+$pf-global--breakpoint--2xl: 1450px !default;
```

Comment:
We should be able to import them from vendor, see:
https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/scss/variables.scss
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/AutoComplete/auto-complete.scss#L1

Reply:
nice thanks! I will change it.
while we are at it, is it possible to make those variables available all over the app without the import?
can we do the same for colors variables?